res/swappy.glade

```diff
@@ -752,6 +757,22 @@
                 <property name="always_show_image">True</property>
                 <signal name="clicked" handler="save_clicked_handler" swapped="no"/>
               </object>
+              <packing>
+                <property name="expand">False</property>
+                <property name="fill">True</property>
+                <property name="position">1</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkButton" id="save_as">
+                <property name="visible">True</property>
+                <property name="can-focus">False</property>
+                <property name="receives-default">True</property>
+                <property name="tooltip-text" translatable="yes">Save Surface As</property>
```

Comment:
You are adding translatable content. You need to run `ninja -C build swappy-update-po` and commit the modified `.po` files.